components/dialog/dialog-fullscreen.js

```diff
@@ -200,6 +210,16 @@ class DialogFullscreen extends LocalizeCoreElement(AsyncContainerMixin(DialogMix
 
 	render() {
 
+		const heightOverride = {} ;
+		if (this._ifrauContextInfo) {
+			this._inIframe = true;
```

Comment:
It's not great to be setting properties from inside `render()`, since this is going to trigger an unnecessary re-render afterwards.

One option would be to move the `_inIframe` into the dialog mixin and set it to `true` when `_ifrauContextInfo` gets set. That works pretty well actually since the normal dialog needs it as well and the property is duplicated there.

Reply:
Oh and actually a good side effect of that will be that when the `_ifrauContextInfo` connects, it'll set `_inIframe` to `true` which will cause a re-render. As it stands now, technically I think it could connect at a random time and nothing would cause a re-render for these overrides to be set.